### DISCUSSION TITLE
GitHub change of policy for supporters

### DISCUSSION THREAD

**Discussion by STyx2909** [Vdauphin/HeartsAndMinds#1480] [General]
@Vdauphin I am not going to provide GitHub with my Card details. Is there an other way to keep supporting you?

**Comment by Vdauphin** [comment]
Hello,
https://www.paypal.me/Vdauphin
Cheers

**Reply by STyx2909** [reply]
> Hello, https://www.paypal.me/Vdauphin Cheers

Solid copy. Delivered. ;-)
